packages/framework/src/Form/Admin/Navigation/NavigationItemFormType.php

```diff
@@ -23,14 +23,14 @@
 class NavigationItemFormType extends AbstractType
 {
     /**
-     * @var \App\Model\Category\Category[]
+     * @var \Shopsys\FrameworkBundle\Model\Category\Category[]
      */
     private array $categoryPaths;
```

Comment:
Should be protected, this applies to several occurences in this class

Reply:
actually, this is a form type and these are not supposed to be extended by inheritance :slightly_smiling_face: